src/FOHInterface/FOHController.java

```diff
@@ -28,4 +32,74 @@ public void markItemUnavailable(int itemID) {
     public Object[][] getOrders() {
         return null;
     }
+
+    //Methods
+
+    public Customer createCustomer(int id, String name, String[] allergens, String[] disabilities){
+        //this makes a customer class.
+        return new Customer(id, name, new BookingCollection(), allergens, disabilities);
+        //TODO: Send this information to the Database to store
+    }
+
+    public BookableTable createBookableTable (int tableID, int capacity, boolean isAccessible){
+        //this makes a Bookable table class
+        return new BookableTable(tableID, capacity, isAccessible);
+        //TODO: Send this information to the Database to store
+    }
+
+    public Waiter createWaiter(int staffID, String name, int pin, String waiterType, String email){
+        //This make the waiter class
+        return new Waiter(staffID, name, pin, waiterType, email);
+        //TODO: Send this information to the Database to store
+    }
+
+    public PhysicalTable createTable(int tableID, int childSeats){
+        //THis makes a physical table class
+        return new PhysicalTable(tableID, childSeats);
+        //TODO: Send this information to the Database to store
+    }
+
+
+    public void makeBooking(Customer customer, int bookingID, BookableTable reservedTable, String note, int numberOfDiners, long time){
+        //This method creates a booking class and then is added in the booking collection for its respectable Bookable table and customer
+
+        if (!reservedTable.checkAvailability(time)){
+            //TODO: Indicate that there is no availability for this table at the given time.
+        }
+        else if (!reservedTable.isAccessible()){ //TODO: add conditions in the IF statement based on customer disabilities
```

Comment:
Some customers won't need an accessible table.

Reply:
This if statement is for those customers who do need an accessible table, this will be indicated in their disability.